modules/objc/pages/_partials/pn-issues-list.adoc

```diff
@@ -1,5 +1,40 @@
 // BEGIN: tagged-inclusion -- issues-list for Objective-C
 
+=== 2.8.4
+// tag::issues-2-8-4[]
+==== Deprecated
+// tag::deprecated-2-8-4[]
+// end::deprecated-2-8-4[]
+
+==== Enhancements
+// tag::enhancements-2-8-4[]
+include::{root-partials}pn-issues-list-litecore.adoc[tag=enhancements-2-8-4]
+
+* https://issues.couchbase.com/browse/CBL-1474[CBL-1474] -- https://issues.couchbase.com/browse/CBL-1474[Support Apple Silicon (M1)]
+* https://issues.couchbase.com/browse/CBL-1580[CBL-1580] -- https://issues.couchbase.com/browse/CBL-1580[Replication Heartbeat]
+* https://issues.couchbase.com/browse/CBL-1601[CBL-1601] -- https://issues.couchbase.com/browse/CBL-1601[Replication Heartbeat Unit tests ]
+
+// end::enhancements-2-8-4[]
+
+==== Fixed
+// tag::fixed-2-8-4[]
+include::{root-partials}pn-issues-list-litecore.adoc[tag=fixed-2-8-4]
+
+* https://issues.couchbase.com/browse/CBL-399[CBL-399] -- https://issues.couchbase.com/browse/CBL-399[CouchbaseLite Mac Catalyst compatible build]
+* https://issues.couchbase.com/browse/CBL-1477[CBL-1477] -- https://issues.couchbase.com/browse/CBL-1477[AcceptOnlySelfSigned may accept non-self-signed certificates]
+* https://issues.couchbase.com/browse/CBL-1570[CBL-1570] -- https://issues.couchbase.com/browse/CBL-1570[Discussion on `.framework` vs `.xcframework` ]
+* https://issues.couchbase.com/browse/CBL-1602[CBL-1602] -- https://issues.couchbase.com/browse/CBL-1602[Cherry pick 2.8.4 release items]
+* https://issues.couchbase.com/browse/CBL-1606[CBL-1606] -- https://issues.couchbase.com/browse/CBL-1606[Verify 2.8.4 binary before QE handover]
+* https://issues.couchbase.com/browse/CBL-1610[CBL-1610] -- https://issues.couchbase.com/browse/CBL-1610[Update the Jenkins to test all Mac variant]
+// end::fixed-2-8-4[]
```

Comment:
The last four issues are internal tasks.

Reply:
Dropped CBL-1570, 1602, 1606, 1610